src/components/SamlProviderConfiguration/index.jsx

```diff
@@ -34,7 +35,7 @@ class SamlProviderConfiguration extends React.Component {
       .catch((error) => {
         const errorMsg = error.message || error.response.status === 500
           ? error.message : JSON.stringify(error.response.data);
-        NewRelicService.logAPIErrorResponse(errorMsg);
+        logError.logAPIErrorResponse(errorMsg);
```

Comment:
Is this supposed to be just `logError` vs `logError.logAPIErrorResponse`?

Reply:
We found that `logAPIErrorResponse` does not actually exist in `frontend-platform`, so this is what we get. :)